test/kubemark/start-kubemark.sh

```diff
@@ -225,6 +225,22 @@ function start-hollow-nodes {
   wait-for-hollow-nodes-to-run-or-timeout
 }
 
+# Annotates the node objects in the kubemark cluster to make their size
+# similar to regular nodes.
+function resize-node-objects {
```

Comment:
I already mentioned it in Matt`s WIP PR - I can live with it as a temporary thing, but I would like explicit TODO and issue filed to replace it with image-preloading that is in progress in CL2:
https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/pkg/imagepreload/imagepreload.go

Reply:
Yes, please add a TODO and file an issue linked to that todo to replace this temporary mechanism with imagepreload once it supports kubemark clusters

Reply:
Done. I created an issue in #90833 which mentions this PR.

Reply:
Please replace with:

```
TODO(#90833): Replace this with image preloading ... 
```

Reply:
Done as well ;).